src/Conversion/KrnlToLLVM/CMakeLists.txt

```diff
@@ -6,9 +6,11 @@ add_onnx_mlir_library(OMKrnlToLLVM
   LINK_LIBS PUBLIC
   OMSupport
   MLIRAffineToStandard
+  MLIRMemRefToLLVM
```

Comment:
When I imported the mlir.cmake file, I tried to limit the libraries that things get linked against to the fewest possible. One thing we could do if we want to avoid some of the churn here is link against *all* conversion libraries. I think it's better to keep a limited list, but it is more work.

Reply:
@sstamenova I added the lib here because MemRef was split away from Standard, it now exists as its own lib. Without it here, I get a missing code error when linking. I don't have issue with keeping a limited list at this time. Hopefully these "splitting the libs" will remain rare events. However, if you are inclined to make a link against all conversion libs, I have no problem with that, but in a subsequent PR.

Reply:
No, I like the short list and I don't think linking against all the conversion libs is necessary. But since I saw that you had to add a couple of them, I thought I'd offer it as an option if you are so inclined.